docs/src/main/sphinx/connector/postgresql.rst

```diff
@@ -46,6 +46,53 @@ determine the user credentials for the connection, often a service user. You can
 use :doc:`secrets </security/secrets>` to avoid actual values in the catalog
 properties files.
 
+SQL support
+-----------
+
+The connector provides **read access and limited write access** to data and
+metadata in Delta Lake. The following section lists all supported SQL statements
+and provides further details.
+
+.. important::
+
+    No other SQL statements are supported.
+
+Read access:
+
+* :doc:`/sql/match-recognize`
+* :doc:`/sql/select`
+* :doc:`/sql/show-catalogs`
+* :doc:`/sql/show-columns`
+* :doc:`/sql/show-schemas`
+* :doc:`/sql/show-tables`
+* :doc:`/sql/describe`
+
+Write access:
+
+* :doc:`/sql/alter-schema`
+* :doc:`/sql/alter-table`
+* :doc:`/sql/create-schema`
+* :doc:`/sql/create-table`
+* :doc:`/sql/create-table-as`
+* :doc:`/sql/delete`
+* :doc:`/sql/drop-schema`
+* :doc:`/sql/drop-table`
+* :doc:`/sql/insert`
+* :doc:`/sql/show-create-schema`
+* :doc:`/sql/show-create-table`
+* :doc:`/sql/update`
```

Comment:
Not supported.

Reply:
DELETEs are supported but not UPDATEs.

Reply:
You said, previously, DELETE was not supported, along with other statements, in the list.  Was that just a mistake?

Reply:
DELETE support got added in the meantime for all (almost?) JDBC connectors. 🎉

Reply:
Nice.